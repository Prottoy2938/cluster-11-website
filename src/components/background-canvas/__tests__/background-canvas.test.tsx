import React from "react";
import BackgroundCanvas from "../background-canvas";
import renderer from "react-test-renderer";

function createNodeMock(element) {
  if (element.type === "input") {
    return {
      focus() {},
    };
  }
  return null;
}

it("renders correctly", () => {
  const options = { createNodeMock };
  const tree = renderer.create(<BackgroundCanvas />, options);
  expect(tree).toMatchSnapshot();
});
