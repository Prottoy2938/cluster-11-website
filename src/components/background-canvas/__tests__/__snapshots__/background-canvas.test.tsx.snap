// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take and match snapshot 1`] = `
<DocumentFragment>
  <div>
    <canvas
      class="canvas"
    />
  </div>
</DocumentFragment>
`;
